### PR TITLE
Create new miniforge Blogpost

### LINKED ISSUE #68: installing mambaforge: tell people to scroll down the license agreement
In the post about mambaforge, some people are confused by the license message looking different than in the screenshot. Tell them to scroll down.

### LINKED ISSUE #73: Update mamba blog post to recommend miniforge instead of mambaforge
`mambaforge` is discontinued and it is integrated into `miniforge` now (check message below from miniforge Github repository):

<details>
  <summary>Click to see message</summary>
Mambaforge (Discouraged as of September 2023)
With the [release](https://github.com/conda-forge/miniforge/releases/tag/23.3.1-0) of Miniforge3-23.3.1-0, that incorporated the changes in https://github.com/conda-forge/miniforge/pull/277, the packages and configuration of Mambaforge and Miniforge3 are now identical. The only difference between the two is the name of the installer and, subsequently, the default installation directory.

Given its wide usage, there are no plans to deprecate Mambaforge. If at some point we decide to deprecate Mambaforge, it will be appropriately announced and communicated with sufficient time in advance.

As of September 2023, the new usage of Mambaforge is thus discouraged. Bug reports specific to Mambaforge will be closed as won't fix.

</details>

The link should still be this one: https://github.com/conda-forge/miniforge#miniforge3

But we should point to download and install `miniforge` instead of `mambaforge`. The rest can remain the same, `mamba` should work from `miniforge`.

### PR DESCRIPTION
**Pull request by marabuuu**
Hi @stefanhahmann,

Thank you for your input regarding this blog post. I decided to go with your suggestion

> Perhaps, the cleanest solution would have been to actually write a new blog post (like we did, to document the changes from anaconda to mambaforge) and reference the new blog post from the current one (like the mambaforge blog post was referenced from the anaconda blog post).

Because if we change the link then we would need to update all the resources where we mentioned this blogpost and I think this installation blogpost was mentioned in several different resources, not only my own, but also for example Roberts and I would not have the overview in which of these resources I need to update the link.

Therefore, I created here a new PR as this was easier for me than modifying PR #79. Hope that is oki. If we merge this one, I will delete the other PR.

It would be cool if you could check if the link in the outdated blog post works for you that should link to the new blog post (https://biapol.github.io/blog/mara_lampert/getting_started_with_miniforge_and_python/Readme.html).

**Just for documentation purposes**

I updated the Miniforge blogpost again and it would be super cool if you could have a look at it. 
I had a chat with Robert and there I found out that we don't need to write `mamba ...` anymore, but can simply use `conda ...`. 

And now, I was trying to find out if I can also remove the `-c conda-forge` from all commands and it seems to be ok, because according to the [conda-forge documentation](https://conda-forge.org/docs/user/introduction/), conda-forge is the default channel of Miniforge.

I feel that these two changes make the installation instructions a bit easier to follow for python beginners, but I am always happy about your input and let me know if you want to have anything changed.

And I just saw that two Issues are still open regarding this blogpost that you Stefan and also Till already addressed, so I will add them here so they are closed when we merge this:
closes #73 
closes #68 

Best,
Mara

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ If you have feedback or suggestions, if spotted a typo, broken link or misleadin
 
 ## Recent blog posts
 
+### [Getting started with Miniforge and Python](mara_lampert/getting_started_with_miniforge_and_python/readme)]
+
+<small>[Mara Lampert](mara_lampert/readme), July 8th, 2024</small><br>
+<img src="images/package_manager.jpeg" width="100" align="right"><p>This post will help you to get started with Python using Miniforge. More precisely, you will learn how to install Miniforge, how to create and use conda environments and know about some very important packages. </p>
```
Comment (stefanhahmann) [inline docs/intro.md:16]:
```suggestion
<img src="images/package_manager.jpeg" width="100" align="right"><p>This post will help you to get started with Python using Miniforge. More precisely, you will learn how to install Miniforge, how to create and use conda environments and you will get to know some important packages for bio image analysis.</p>
```

File: docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
+# Getting started with Miniforge* and Python
```
Comment (stefanhahmann) [inline docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md:1]:
```suggestion
# Getting started with Miniforge and Python 
```

File: docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
+# Getting started with Miniforge* and Python 
+[Mara Lampert](../readme), July 8th, 2024
+
+> **_Note:_** `(*)` An older version of this blog post covered Mambaforge. After the release of Miniforge 23.3.1 in August 2023, Miniforge and Mambaforge are essentially identical. The only difference is the name of the installer and subsequently the default installation path. In the current version of this blog post, we describe Miniforge3.
```
Comment (stefanhahmann) [inline docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md:4]:
```suggestion
```

File: docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+## Introduction to Python and Miniforge 
+This blog post explains what Python and [Conda](https://docs.readthedocs.io/en/stable/guides/conda.html) / Miniforge is, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 
+
+> **_Note:_** This is an update of a [previous Blogpost](https://biapol.github.io/blog/johannes_mueller/anaconda_getting_started/Readme.html) written by [Johannes](https://biapol.github.io/blog/johannes_mueller/Readme.html).
```
Comment (stefanhahmann) [inline docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md:9]:
```suggestion
> **_Note:_** This is an update of a [previous Blogpost](https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python/readme.html). 
```

File: docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+
+
+**Why do we need Conda to use Python?**
+
+__Python__ is a programming language which is easy to learn and very important in scientific data analysis nowadays. 
+
+__Conda__ is a __package manager__ which can be used with Python. It is a software allowing to install other software. Read more about Conda [here](https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/). 
+
+## Installation of Miniforge 
+Here, I am going to show how to install Miniforge. It comes with everything you need and downloads it from a community-driven open source software provider called  [conda-forge](https://conda-forge.org/).
```
Comment (stefanhahmann) [inline docs/mara_lampert/getting_started_with_miniforge_and_python/readme.md:19]:
```suggestion
Here, I am going to show how to install Miniforge. It comes with everything you need and downloads packages from a community-driven open source software provider called  [conda-forge](https://conda-forge.org/). 
```

**Review by stefanhahmann — CHANGES_REQUESTED** [review]
Looks mostly good to me. I have suggested some minor changes below.

The link 
https://biapol.github.io/blog/mara_lampert/getting_started_with_miniforge_and_python/Readme.html looks good to me. Whether it actually works, I can only check, once this PR is merged.

**Comment by stefanhahmann** [comment]
When this PR is merged, https://github.com/BiAPoL/blog/pull/80 can be closed.

**Comment by marabuuu** [comment]
Hi @stefanhahmann,

Thank you for your help and input. I hope I addressed everything now and I think we can merge this now.

Best,
Mara

**Review by stefanhahmann — APPROVED** [review]

**Comment by stefanhahmann** [comment]
Looks alright. Feel free to merge.

**Pull request merged**